corehq/apps/app_manager/templates/app_manager/v2/partials/case_list.html

```diff
@@ -76,6 +79,34 @@ <h4 class="panel-title panel-title-nolink">
                 </label>
             </div>
         </div>
+        {% endif %}
+
+        {% if request|feature_preview_enabled:"SHARE_CASE_TILE_CONTEXT" %}
+        {{ request|toggle_tag_info:"CASE_LIST_TILE" }}
+        {% if request|toggle_enabled:'CASE_LIST_TILE' %}
+        {% if request|toggle_enabled:'SHOW_PERSIST_CASE_CONTEXT_SETTING' %}
```

Comment:
i am not sure if that would be good to do. Those two toggles might be used by different set of domains.

Though just the check for `SHOW_PERSIST_CASE_CONTEXT_SETTING` is sufficient but if somebody enables that without enabling `CASE_LIST_TILE`, this might give unexpected results in case the case tiles are configured and then the `CASE_LIST_TILE` was removed.